site/content/docs/multistep-checks/_index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Getting started
+weight: 14
+slug: /
+menu:
+  resources:
+    parent: "Multistep API checks"
+    identifier: "multistep-checks"
+aliases:
+    - /docs/multistep-checks/quickstart/
+    - /docs/multistep-checks/getting-started/
+cli: true
+---
+
+This guide gives you all the info to create your first Multistep API check with Checkly. You should have some prior
+knowledge of working with Javascript and/or Node.js.
+
+## What is a Multistep API check?
+
+A Multistep API check allows you to write Node.js scripts that can run multiple API requests in sequence, with arbitary code between requests. This allows you to monitor entire API user flows with a single check, working with response data in between requests exactly as a user of your API would to ensure that these interactions result in the correct results.
+
+Examples of API squences might be:
+* Users can authenticate and access restricted data
+* Users can get, set and remove data from their account
+* Users can add items to a shopping card, checkout and go through the payment flow.
+
+Monitoring your API user flows instead of individual endpoints gives confidence that your product as a whole work as intended and that expected interactions between API calls are functional.
```

Comment:
```suggestion
Monitoring your API user flows instead of individual endpoints gives confidence that your product as a whole work as intended and that the expected interactions between API calls are functional.
```